website/apps/transcripts/templatetags/missionstatic.py

```diff
@@ -2,14 +2,11 @@
 from django import template
 from django.conf import settings
 from django.contrib.staticfiles.storage import (
-    get_storage_class, staticfiles_storage,
+    storages, staticfiles_storage,
 )
 
 
-staticfiles_digest_free_storage = get_storage_class(
-    settings.STATICFILES_DIGEST_FREE_STORAGE,
-)()
-
+staticfiles_digest_free_storage = storages["staticfiles"]
```

Comment:
Erm, this is no longer a digest free storage? It's now using whitenoise when previously we separated this storage from the default storage deliberately - although I can't remember the details. The commit message should explain why this change is happening because it's a more subtle one. Ideally it would reference back to the commit that introduced the separation to enable review without manual spelunking.

Reply:
(Beyond that LGTM.)

Reply:
This is a great spot, thanks @jaylett! I don't think I'd clocked the difference because I was mostly just trying to get to a state where I could run the site, sort of thing.

I've reinstated the correct behaviour in be706b33da11ec8a22a623baae843c68a9219a75 (spotting issues starting Gunicorn and serving CSS in production along the way, resolved in d88f1ed60adde2e1fb41c693b307f47b4affe7e8 and da4829dbe9bfeb05b326deb507da1b8012530e15 respectively), and will explain the change better in the resultant commit, linking to 0fa3926da8b98932418de413e1cc13d158f85936 (where digest-free statics were introduced) as you suggest.